### ISSUE TITLE
Tasks in Nextcloud Notification

### ISSUE THREAD

**Issue by Leprazoriy** [nextcloud/server#23594]
Is it possible to make the user receive a notification when a new task appears.
![Notification](https://user-images.githubusercontent.com/72513510/96623136-93ce2880-1313-11eb-8f09-ac3cf56dbbab.jpg)
I have enabled calendar notifications in settings:
![Notification1](https://user-images.githubusercontent.com/72513510/96623026-62edf380-1313-11eb-9cad-8ed1084093be.jpg)
I also installed the app "Notifications for calendar event updates", but notifications are received only for events created in the Calendar application, and notifications about new tasks are not received.

**Comment by raimund-schluessler** [comment]
Regarding the "Notifications for calendar event actions" app: This app is only about events (as the name states 😉). I don't know if it makes sense to app "task actions" as well or if this should rather be a separate app. @nickvergessen What do you think?

**Comment by nickvergessen** [comment]
Well it depends, if the information the activity is showing is good enough, that should be used. If you want e.g. actions to "delay" a task, or "resolve it" from the notification (sounds handy?) you need to implement notifications in the tasks app directly.

**Comment by raimund-schluessler** [comment]
> Well it depends, if the information the activity is showing is good enough, that should be used. If you want e.g. actions to "delay" a task, or "resolve it" from the notification (sounds handy?) you need to implement notifications in the tasks app directly.

As far as I understood it, for events it is part of server?! Shouldn't we handle tasks just the same then and implement it in server (see screenshot in first post)?

And your app https://github.com/nickv-nextcloud/event_update_notification is also standalone and not included in Calendar.

**Comment by nickvergessen** [comment]
> As far as I understood it, for events it is part of server?! Shouldn't we handle tasks just the same then and implement it in server (see screenshot in first post)?

Well but tasks can be "resolved" and so on, so that would be a handy "action" for the notifications. but that is only possible when you use notifications directly and not via activities

**Comment by raimund-schluessler** [comment]
Yes, I think we should use notifications and not activities. But my question is if this shouldn't be part of server (as it seems to be for events)?

**Comment by nickvergessen** [comment]
Hmm no strong feelings on my side, but I always prefer things to be in the apps, to keep the size maintainable.
Also you can then link to the correct page of the task app, while when it would be in server we couldn't link as tasks might not be installed.

**Comment by raimund-schluessler** [comment]
> Also you can then link to the correct page of the task app, while when it would be in server we couldn't link as tasks might not be installed.

Fine with me to have it in the Tasks app, but we do link already to the Tasks app in server for todos found by the unified search, so I think this shouldn't be an issue.
